raiden_contracts/data/source/raiden/TokenNetworkRegistry.sol

```diff
@@ -42,20 +41,18 @@ contract TokenNetworkRegistry is Utils, Controllable {
     /// MAX_UINT256 means no limits
     constructor(
         address _secret_registry_address,
-        uint256 _chain_id,
+        uint256 _chain_id,  // Ignored and only left for backward compatibility
```

Comment:
I see the point of leaving this in, but I'd rather remove it as the adaptations for this should be small.